apps/web/configuration/tailwind.config.ts

```diff
@@ -2,15 +2,22 @@ import sfTypography from '@storefront-ui/typography';
 import { tailwindConfig } from '@storefront-ui/vue/tailwind-config';
 import type { Config } from 'tailwindcss';
 import defaultTheme from 'tailwindcss/defaultTheme';
-import { fontFamilyDisplay, fontFamilyText } from './fontFamily.config';
+import { fontFamilyText } from './fontFamily.config';
 
 export default {
   presets: [tailwindConfig],
   content: ['./**/*.vue', '../../node_modules/@storefront-ui/vue/**/*.{js,mjs}'],
   theme: {
     extend: {
+      sfTypography: () => ({
+        'display-1': {
+          fontFamily: 'inherit',
+        },
+        'display-2': {
+          fontFamily: 'inherit',
+        },
+      }),
```

Comment:
**question**: I'm guessing this means `display-1` and `display-2` will use the body font? Given that the display font is only used in one place, wouldn't it be better to just replace the class there?